samples/ClientSample/ClientSample.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\..\build\dependencies.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
How does this compile without NS.Library.NETFramework? I think we need that reference here.